diffs/AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_after.md

```diff
@@ -0,0 +1,74 @@
+## Reserve changes
+
+### Reserve altered
+
+#### m.USDC ([0xEA32A96608495e54156Ae48931A7c20f0dcc1a21](https://andromeda-explorer.metis.io/address/0xEA32A96608495e54156Ae48931A7c20f0dcc1a21))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x3B88f83B2b0444F14B6fc904040F648206F551fc](https://andromeda-explorer.metis.io/address/0x3B88f83B2b0444F14B6fc904040F648206F551fc) | [0x3c8DF161A59F2a3ed4D40F073395d6a0f14290C0](https://andromeda-explorer.metis.io/address/0x3c8DF161A59F2a3ed4D40F073395d6a0f14290C0) |
+| variableRateSlope1 | 4 % | 5 % |
```

Comment:
we don't have interest rate parameters ingested for metis so i can't use bigquery to confirm whether the before value is correct